### PR TITLE
Fix codecov and GemRequirementTest failing

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by sils1297** [comment]
fix it? Really? :( https://github.com/coala-analyzer/coala/pull/2500/commits/66ef45dd73f8e2c7b8a4f956dff27695454b5846 is ready

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2500/commits/86b244ad113c6c08752bb10c427398ef0d92a2ca

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2500/commits/40821af5eaa234139fa7f404d2df2560f74828eb

**Comment by Adrianzatreanu** [comment]
@sils1297 idea about coverage? not sure if its from my commit ..

**Comment by Adrianzatreanu** [comment]
well apparently coverage dropped on GemRequirement. because tests are being skipped on circleCI i think?? not sure, do we add pragma no cover?

**Comment by sils1297** [comment]
but they shouldn't be skipped right? They're passing right?

**Comment by Adrianzatreanu** [comment]
@sils1297 yes, but yet no coverage :(

File: tests/bears/requirements/GemRequirementTest.py [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+from coalib.misc.Shell import call_without_output
 
+cmd = ['gem', 'list', '-i', 'ruby']
+if platform.system() == 'Windows':  # pragma: no cover
+    cmd = ['cmd', '/c'] + cmd
 
-@unittest.skipIf(shutil.which('gem') is None, "Gem is not installed.")
+
+@unittest.skipIf(shutil.which('gem') is None or
+                 bool(call_without_output(cmd)),
```
Comment (Redridge) [inline tests/bears/requirements/GemRequirementTest.py]: why is this line useful? this is the one breaking the coverage

**Comment by Adrianzatreanu** [comment]
reack 9aa8ea5

**Comment by Redridge** [comment]
reack 1c3d6d24bf761ab8448bd26684116fb102acc039

**Comment by Redridge** [comment]
@rultor merge

**Pull request merged**